### DISCUSSION TITLE
how to send a instant messages(text message) from the command line

### DISCUSSION THREAD

**Discussion by colorhacker** [baresip/baresip#2663] [Q&A]
Hello, I am a novice. I am currently running baresip directly under Linux. I can make calls and receive instant messages (text messages). How can I send text messages through the command line? I use MicroSIP to send to baresip under Linux.

**Comment by Tybil** [comment]
Hello, you need to create a contact(put it in the contact file) for your destination to whom you want to send your message, switch to the contact with `>` or `<`. You can then use the `/message` command to send a message to your current contact.

**Comment by colorhacker** [comment]
thank you so much!
It is necessary to add the user group that needs to be contacted in the configuration file, and then select the user( switch to the contact with > or <) in the command line, and then enter /message or capital M.
